hackclub.com.yaml

```
@@ -1473,6 +1473,10 @@ help.bank:
 help.hcb:
   type: CNAME
   value: hcb.helpscoutdocs.com.
+hersey:
+  - ttl: 600
+    type: CNAME
+    value: hersey-hack-club.vercel.app/.
```

DNS records can't have paths `/`


```suggestion
    value: hersey-hack-club.vercel.app.
```